### PR TITLE
Add MergeTags in Link component attribute

### PR DESCRIPTION
**Pull request by alexmateos**
Created this PR to add merge tags in Link component attribute and fix this issue: https://github.com/zalify/easy-email-editor/issues/195

Before:

image

After:

image

### CODE REVIEW

**Review by m-Ryan — CHANGES_REQUESTED** [review]

File: packages/easy-email-extensions/src/AttributePanel/components/attributes/Link.tsx [hunk 1]
```diff
@@ -1,23 +1,49 @@
 import React, { useMemo } from 'react';
-import { useFocusIdx } from 'easy-email-editor';
+import { useFocusIdx, IconFont } from '@liveconnect/easy-email-editor';
 import { IconLink } from '@arco-design/web-react/icon';
```
[the remark this answers is not in the document]
Comment (alexmateos) [reply packages/easy-email-extensions/src/AttributePanel/components/attributes/Link.tsx]: Sorry, messed up with the copy paste. Fixed it!

**Review by m-Ryan — APPROVED** [review]

**Comment by m-Ryan** [comment]
@alexmateos  released,  please update to th latest version
 - easy-email-core@4.16.0
 - easy-email-editor@4.16.0
 - easy-email-extensions@4.16.0

**Comment by alexmateos** [comment]
image

Your builds are failing since you added pnpm. Actually it fails in local if you dont install pnpm.

You should check that out

**Comment by m-Ryan** [comment]
The test has been fixed

**Pull request merged**